packages/authgear-react-native/src/index.ts

```diff
@@ -908,6 +967,16 @@ export class ReactNativeContainer {
     }
   }
 
+  /**
+   * Enable biometric authentication for the signed in user.
+   * Platform specific biometric authentication UI will be shown.
+   * You may first enable biometric authentication in Authgear Portal (Your Project \> Authentication \> Biometric \> Enable biometric authentication).
+   * Otherwise, `BiometricDisallowed` error will be thrown.
```

Comment:
We do not have `BiometricDisallowed` error. Remove this line.

Reply:
ok